### PR TITLE
Add New Token BSAMO

### PR DESCRIPTION
**Pull request by buffsamo**
## I agree to not ping _anybody_ on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.

---

PRs are reviewed in bulk and and can take up to **two weeks** to be merged.

_This repository is managed using an auto merge action. Please ensure your PR has no deleted lines, and it will be merged._

## **Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR.
DON'T modify any other token on the list.

At minimum each entry should have

- Token Address: 2XSuy8RSESbtYRBbVHxGWuoikn3B6iXKVKzN4i3owTCf
- Token Name: BUFF SAMO
- Token Symbol: BSAMO
- Logo: assets/mainnet/2XSuy8RSESbtYRBbVHxGWuoikn3B6iXKVKzN4i3owTCf/logo.png
- Link to the official homepage of token: https://www.buffsamo.tech/

### CODE REVIEW

**Comment by coel00** [comment]
looks very nice and cool someday it will be a big and successful project !!!!

**Comment by buffsamo** [comment]
How to delete banana bucks?

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -624,19 +624,19 @@
...
-        "utility-token"
-      ],
-      "extensions": {
-        "website": "http://bananabucks.io/"
-      }
+    {
+      "chainId": 101,
+      "address": "2Dzzc14S1D7cEFGJyMZMACuoQRHVUYFhVE74C5o8Fwau",
+      "symbol": "BAB",
+      "name": "Banana Bucks",
```
Comment (buffsamo) [inline src/tokens/solana.tokenlist.json:631]: Delete this please

**Comment by buffsamo** [comment]
Done

**Pull request closed without merging**